crawler/constants.py

```diff
@@ -261,9 +261,22 @@
 RABBITMQ_CREATE_FEEDBACK_ORIGIN_ROOT = "root"
 RABBITMQ_CREATE_FEEDBACK_ORIGIN_SAMPLE = "sample"
 
+RABBITMQ_FIELD_COG_UK_ID = "cogUkId"
```

Comment:
Advisory: I have probably said this before but seeing lots of constants which look like they are related makes me think that they could be in an object. It's difficult for me to say if that is the best option based on looking at the code but these fields are grouped together and passed around a fair bit. I can see areas in the code which may benefit from this approach with a bit of encapsulation.

Reply:
Good shout.  I was copying the existing pattern here, but I think grouping these keys would be a good idea.  Unfortunately if you meant using objects instead of dictionaries to contain the message contents I'm not sure if that's going to work as the `fastavro` library gives you a plain dict object after decoding.  However I can look at this again.  I wasn't having any luck when I tried it before.

Reply:
I think it would be worth doing this as a refactor PR rather than muddying the waters on this one so I'll have a trial of that in another branch.